Kitodo/src/main/java/org/kitodo/production/services/data/LdapServerService.java

```diff
@@ -162,6 +162,11 @@ public void createNewUser(User user, String password)
             return;
         }
 
+        if ( user.getLdapGroup().getLdapServer() == null) {
```

Comment:
We use `Objects.isNull()` for nullChecks (see 5 lines above). It's a convention, and I would like to keep it consistent.
If this is a good idea can be discussed here: #4087